app/navigation/nonsipp/SharesDisposalNavigator.scala

```diff
@@ -60,8 +60,12 @@ object SharesDisposalNavigator extends JourneyNavigator {
       }
 
     case WhenWereSharesSoldPage(srn, shareIndex, disposalIndex) =>
+      controllers.nonsipp.sharesdisposal.routes.HowManySharesSoldController
+        .onPageLoad(srn, shareIndex, disposalIndex, NormalMode)
+
+    case HowManySharesSoldPage(srn, shareIndex, disposalIndex) =>
       controllers.routes.UnauthorisedController.onPageLoad()
-//      controllers.nonsipp.sharesdisposal.routes.HowManySharesSoldController
+//      controllers.nonsipp.sharesdisposal.routes.TotalConsiderationSharesSoldController
```

Comment:
One option in cases like this @ccatosdevelopment is to keep the redirection to unauthorised and not add the commented out section.  Then later on when the new page is added, change the navigation to redirect to the right page. I.e. not to work ahead of the ticket.